### ISSUE TITLE
Empire being overwritten when it shouldn't

### ISSUE THREAD

**Issue by hagamablabla** [wyandotte/stellaris-empire-designer#1]
When I save an empire, it overwrites another with a different name.
Not really sure what information I should provide, so please ask for anything that could help.

**Comment by wyandotte** [comment]
Does that persist when you reload the Saved Empires page? Saving with localStorage is sometimes a bit wonky, but it could very well be a bug in the code.

**Comment by hagamablabla** [comment]
Yup, I've closed and reopened the page multiple times, but the overwritten empire never comes back. I've noticed that it starts happening after I have five empires saved, and it always saves to the third empire in the list.

**Comment by wyandotte** [comment]
Hmmm, could be a quirk of localStorage. Maybe it exceeds the maximum allowed capacity. Which browser do you use? I will try to reproduce the issue.
